numerapi/numerapi.py

```diff
@@ -1103,3 +1103,19 @@ def daily_submissions_performances(self, username: str) -> List[Dict]:
         performances = [p for p in performances
                         if any([p['correlation'], p['fnc'], p['mmc']])]
         return performances
+
+    def get_dataset_list(self):
+        query = """
+            query { listDatasets }
+        """
```

Comment:
This doesn't need triple-quotes and should go on one line

Reply:
lets get round number and tournament number in here?